package.json

```diff
@@ -101,6 +93,6 @@
     "name": "frost-nav"
   },
   "pr-bumper": {
-    "coverage": 95.24
+    "coverage": 69.61
```

Comment:
Sadly, this is the actual accurate coverage information. The version of `ember-cli-code-coverage` we were using previously doesn't count coverage for modules that weren't loaded in test, so anything we were missing tests for wasn't reported properly. I also moved some `.skip`'d tests up from the `it` to the `describe` to more accurately reflect components we had no tests for.